server/src/database/models/users.js

```diff
@@ -54,5 +57,8 @@ module.exports = (sequelize, DataTypes) => {
       onUpdate: 'CASCADE'
     });
   };
+
+  Users.beforeCreate((user) => user.id = uuidv4());
```

Comment:
Won't it be easier if you do the password hashing here instead of inside the controller? This way even if you use the `Users` model from any part of the application the `beforeCreate` hook still hashes the password field. You can also research on how to ignore fields on get from the the model

Reply:
ok